tests/kola/afterburn/platform-id

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 ## kola:
 ##   platforms: aws gce
-#
-# Make sure afterburn with old ignition.platform.id kernel argument 
-# ec2 and gce works.
-# See https://issues.redhat.com/browse/OCPBUGS-7733
+##   description: Verify afterburn works with kernel argument 
+##     ignition.platform.id=ec2/gce.
```

Comment:
Should we change the `gce` to `gcp`?

Reply:
I'm just curious since you're changing it [here](https://github.com/openshift/os/pull/1258/files#diff-1e22579e7e92e7fab9527febad05e9cccd286aad07e2a247d8c955531cfe3414R3)